lib/weak_swagger_parameters/definitions/parameter.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+module WeakSwaggerParameters
+  module Definitions
+    class Parameter
+      include WeakSwaggerParameters::Definitions::LeafDefinition
+
+      def initialize(location, type, name, description, options = nil)
```

Comment:
Why not pass the default value as `{}` you seem to don't like nil's anyway down the line.